zulipterminal/ui_tools/boxes.py

```diff
@@ -265,7 +265,7 @@ def track_idleness_and_update_status() -> None:
 
         urwid.connect_signal(self.msg_write_box, "change", on_type_send_status)
 
-    def update_recipients(self, write_box: ReadlineEdit) -> None:
+    def update_recipients_from_emails_in_widget(self, write_box: ReadlineEdit) -> None:
```

Comment:
I noticed that this naming using 'in_widget' occurs only in this function in the codebase. Does it actually help us to differentiate it from valid emails, or is it redundant?